knowledge-base/dropdownlist-kb-nodatatext.md

```diff
@@ -0,0 +1,134 @@
+---
+title: No data text in DropDownList popup
+description: Is there a way to have the "No data" text vertically aligned based on the popup height in order to have it always visible?
+type: troubleshooting
+page_title: No data text not visible in DropDownList popup
+slug: dropdownlist-kb-nodatatext
+tags: telerik, blazor, dropdownlist, popup, dropdownlistpopupsettings, css
+ticketid: 1635198
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>DropDownList for Blazor, <br /> AutoComplete for Blazor, <br /> MultiSelect for Blazor, <br /> ComboBox for Blazor, <br /> MultiColumnComboBox for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+
+## Description
+
+When the popup is smaller than the default min-height of the no data template, the "No data" text is not visible.
+
+
+## Cause
+
+The default min-height of the .k-nodata class is 140px. The default min-height of the popup is 200px. If the Height or MaxHeight of the popup is equal to or smaller than 100px - the "No data" text will not be visible.
+
+
+## Solution
+
+The DropDownList popup setting `Class` lets you define a CSS class that will be rendered on the popup element.
```

Comment:
Consider being more specific here. What's the exact solution here? Is it to set the min-height to auto? Is it to set it to a fixed height? Right now this sentence doesn't really provide a solution but shares general information.